Resources/Prototypes/Entities/Objects/Misc/utensils.yml

```
@@ -31,6 +31,7 @@
     - Plastic
     - Trash
   - type: MeleeWeapon
+    animationRotation: 180 # WWDP
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)